### ISSUE TITLE
NTR: CT or MRI: hypertrophy of superior olive (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8909]
New term request CT or MRI: hypertrophy of superior olive (kalia_100531130617):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000848
- label: Superior olive hypertrophy
- parents: [Abnormal pons morphology (HP:0007361)](https://hpo.jax.org/app/browse/term/HP:0007361)
- pmid: 
- def: Abnormal increase in size of the superior olive, also known as the superior olivary complex. The superior olive is a collection of brainstem nuclei that is located in pons, functions in multiple aspects of hearing and is an important component of the ascending and descending auditory pathways of the auditory system.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8909
- synonyms:
